core-addon/Core.js

```diff
@@ -33,7 +33,10 @@ module.exports = class Core {
     this._availableStudies =
       this._fetchAvailableStudies()
           .then(studies => this._updateInstalledStudies(studies));
-
+            // Register a listener to react to remote-settings updates.
+            browser.firefoxPrivilegedApi.onRemoteSettingsSync.addListener((result) => {
+              this._sendStateUpdateToUI(result[0].data);
+          });
```

Comment:
nit: the indentation looks odd here :-)

Reply:
Hm yes my vscode eslint seems to disagree with ours an I haven't figured out why yet, so I have not been using auto-format :( Will figure this out.

Reply:
Oh @Dexterp37 so as I mentioned in a separate comment, I'm pretty sure I'm passing the right data here (the same as `_fetchAvailableStudies()` returns) but I don't see the UI update, I have been focusing on getting all the RS bits right but I haven't figured this out yet.